lib/dashboard.js

```diff
@@ -7,118 +7,127 @@ var StreamView = require("./views/stream-view");
+    });
+  } else {
+    this.views.stdout = new StreamView({
+      parent: this.container,
+      scrollback: this.options.scrollback,
+      events: ["stdout"],
+      color: "green",
+      layoutConfig: config.stdout
+    });
+    this.views.stderr = new StreamView({
+      parent: this.container,
+      scrollback: this.options.scrollback,
+      events: ["stderr"],
+      color: "red",
+      layoutConfig: config.stderr
+    });
+  }
 
-var metricsPosition = function (index, parent) {
-  var top = Math.ceil(index / metricsCount * parent.height);
-  var bottom = Math.ceil((index + 1) / metricsCount * parent.height);
-  return {
-    top: top,
-    height: bottom - top,
-    left: Math.ceil(parent.width * stdOutWidth),
-    width: Math.floor(parent.width * (1 - stdOutWidth))
-  };
-};
+  this.views.cpu = new CpuView({
+    parent: this.container,
+    layoutConfig: config.cpu
+  });
 
-Dashboard.prototype._logStream = function (streamView, data) {
-  var lines = data.replace(/\n$/, "");
+  this.views.eventLoop = new EventLoopView({
+    parent: this.container,
+    layoutConfig: config.eventLoop
+  });
 
-  streamView.onEvent(lines);
-};
+  this.views.memory = new MemoryView({
+    parent: this.container,
+    layoutConfig: config.memory
+  });
 
-Dashboard.prototype._createStdoutView = function (container) {
-  var stdoutView = new StreamView({
-    parent: container,
-    scrollback: this.options.scrollback,
-    label: this.options.interleave ? "stdout / stderr" : "stdout",
-    color: this.options.interleave ? "light-blue" : "green",
-    getPosition: stdOutPosition
+  this.views.help = new HelpView({
+    parent: this.container
   });
 
-  this.eventPump.addListener("stdout", this._logStream.bind(this, stdoutView));
+  this.currentLayout = 0;
 };
 
-Dashboard.prototype._createStderrView = function (container) {
-  if (!this.options.interleave) {
-    var stderrView = new StreamView({
-      parent: container,
-      scrollback: this.options.scrollback,
-      label: "stderr",
-      color: "red",
-      getPosition: stdErrPosition
-    });
-
-    this.eventPump.addListener("stderr", this._logStream.bind(this, stderrView));
-  }
-};
+Dashboard.prototype._configureKeys = function () {
 
-Dashboard.prototype._createMetricsViews = function (container) {
-  _.each(metrics, function (Metric, index) {
-    var view = new Metric({
-      parent: container,
-      getPosition: metricsPosition,
-      index: index
-    });
-    this.eventPump.addListener("metrics", view.onEvent.bind(view));
+  this.screen.key(["left", "right"], function (ch, key) {
+    var delta = key.name === "left" ? -1 : 1;
+    var target = (this.currentLayout + delta + layouts.length) % layouts.length;
+    this._showLayout(target);
   }.bind(this));
-};
 
-Dashboard.prototype._createView = function () {
-  // fixes weird scrolling issue
-  var container = blessed.box({});
+  this.screen.key("?", this.views.help.node.toggle.bind(this.views.help.node));
```

Comment:
small nit: might be slightly easier to read by pulling `this.views.help.node` into a variable